cmd/snap/cmd_prepare_image.go

```diff
@@ -58,8 +61,9 @@ For preparing classic images it supports a --classic mode`),
 			"classic": i18n.G("Enable classic mode to prepare a classic model image"),
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"arch": i18n.G("Specify an architecture for snaps for --classic when the model does not"),
+			"snap": i18n.G("<snap>[=<channel>] specify extra snaps from store or local or optionally control the channel to track for a snap"),
 			// TRANSLATORS: This should not start with a lowercase letter.
-			"extra-snaps": i18n.G("Extra snaps to be installed"),
+			"extra-snaps": i18n.G("Extra snaps to be installed (DEPRECATED)"),
```

Comment:
Slightly curious about this, it seems like --extra-snaps could also accept the new syntax. Is this because the old name (--extra-snaps) is slightly misleading (as it allows to also override existing snaps) and this is a good opportunity to fix this?

Reply:
yes, the old name is misleading so chance to fix it